src/embezzler.ts

```diff
@@ -46,7 +46,17 @@ export class EmbezzlerFight {
   requirements: Requirement[];
   draggable: boolean;
   name: string;
-
+  /**
+   * This is the class that creates all the different ways to fight embezzlers
+   * @classdesc Something goes here
+   * @prop {string} name The name of the fight so you know what you're doing
+   * @prop {boolean} available this should be a statement that checks the availability of the fight
```

Comment:
```suggestion
   * @prop {() => boolean} available Returns whether or not we can do this fight right now.
```
"checks the availability of" feels vague